site/content/in-dev/unreleased/configuring-polaris-for-production.md

```diff
@@ -23,114 +23,179 @@ type: docs
+context resolver by setting the following properties in `application.properties`:
 
-```
-export POLARIS_BOOTSTRAP_CREDENTIALS=my_realm,my-client-id,my-client-secret
+```properties
+polaris.realm-context.realms=POLARIS,MY-REALM
+polaris.realm-context.header-name=Polaris-Realm
 ```
 
-The format of the environment variable is `realm,client_id,client_secret`. You can provide multiple credentials separated by `;`. For example, to provide credentials for two realms `my_realm` and `my_realm2`:
+Where:
 
-```
-export POLARIS_BOOTSTRAP_CREDENTIALS=my_realm,my-client-id,my-client-secret;my_realm2,my-client-id2,my-client-secret2
-```
+- `realms` is a comma-separated list of allowed realms. This setting _must_ be correctly configured.
+  At least one realm must be specified.
+- `header-name` is the name of the header used to resolve the realm; by default, it is
+  `Polaris-Realm`.
 
-It is also possible to use system properties to provide the credentials:
+If a request does not contain the specified header, Polaris will use the first realm in the list as
+the default realm. In the above example, `POLARIS` is the default realm.
 
-```
-java -Dpolaris.bootstrap.credentials=my_realm,my-client-id,my-client-secret -jar /path/to/jar/polaris-service-all.jar bootstrap polaris-server.yml
-```
+### Metastore Configuration
 
-Now, to bootstrap Polaris, run:
+A Metastore Manager should be configured with an implementation that durably persists Polaris
+entities. By default, Polaris uses an in-memory metastore.
 
-```bash
-java -jar /path/to/jar/polaris-service-all.jar bootstrap polaris-server.yml
-```
+> [!WARNING]
+> The default in-memory metastore is not suitable for production use, as it will lose all data
+> when the server is restarted; it is also unusable when multiple Polaris replicas are used.
 
-or in a container:
+To use a durable Metastore Manager, you need to switch to the EclipseLink metastore, and provide
+your own `persistence.xml` file. This file contains details of the database used for metastore
+management and the connection settings. For more information, refer to the [metastore
+documentation]({{% ref "metastores" %}}).
 
-```bash
-bin/polaris-service bootstrap config/polaris-server.yml
+Then, configure Polaris to use your metastore by setting the following properties:
+
+```properties
+polaris.persistence.type=eclipse-link
+polaris.persistence.eclipselink.configuration-file=/path/to/persistence.xml
+polaris.persistence.eclipselink.persistence-unit=polaris
 ```
 
-Afterward, Polaris can be launched normally:
+Where:
 
-```bash
-java -jar /path/to/jar/polaris-service-all.jar server polaris-server.yml
+- `polaris.persistence.type` indicates that we are using the EclipseLink metastore.
+- `polaris.persistence.eclipselink.configuration-file` is the path to the `persistence.xml` file.
+- `polaris.persistence.eclipselink.persistence-unit` is the name of the persistence unit to use (in 
+  case the configuration file has many persistence units).
```

Comment:
Given we've already link to the doc `metastores.md`, I'd recommend to provide an example like the following, instead of descriptions here. But it isn't blocker for me. Let's update the doc ASAP. We can fix it in another PR.
```
./gradlew clean :polaris-quarkus-server:assemble -PeclipseLinkDeps=org.postgresql:postgresql:42.7.4
java -Dpolaris.persistence.type=eclipse-link \
     -Dpolaris.persistence.eclipselink.configuration-file=/path/to/persistence.xml \
     -Dpolaris.persistence.eclipselink.persistence-unit=polaris \
     -jar quarkus/server/build/quarkus-app/quarkus-run.jar
```